### PR TITLE
change readme

### PR DESCRIPTION
**Pull request by onlybaby**
change readme to hello world!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# dummyrepo
+# Dummy Repo
+hello world
```
Comment (yjyao) [inline README.md]: Please correctly capitalize your paragraphs. I.e., change to 

```
Hello world.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# dummyrepo
+# Dummy Repo
```
Comment (yjyao) [inline README.md:1]: Add blank line below titles.

**Review by yjyao — CHANGES_REQUESTED** [review]
Look nice. Just a few nit picks. Good job!

File: README.md (hunk 2 above)
Comment (onlybaby) [reply README.md:1]: Done.

**Comment by onlybaby** [comment]
changes made

**Review by yjyao — APPROVED** [review]

**Pull request merged**